src/PeselValidator/PeselValidator.cs

```diff
@@ -26,13 +28,16 @@ public static bool IsValid(string pesel)
 
         private static bool HasValidDay(string pesel)
         {
+            var b = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'b' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)